web/src/buddy_list.ts

```diff
@@ -325,7 +326,24 @@ export class BuddyList extends BuddyListConf {
         }
     }
 
-    render_section_headers(): void {
+    update_section_headers(): void {
+        const {total_human_subscribers_count, other_users_count} = this.render_data;
+        const formatted_sub_users_count = get_formatted_sub_count(total_human_subscribers_count);
+        const formatted_other_users_count = get_formatted_sub_count(other_users_count);
+        // TODO(evy): what's the correct way of doing this without rerendering anything else?
```

Comment:
Would it be useful to have a template just for the `<h5>` that could then take a parameter for the count? 

I guess the other way around this, since the count is just text (I'm guessing the `formatted_xxx_count` does things like "30K" when appropriate), would be to work in text nodes.

Reply:
But I guess it might also be worth your working on top of #31726? I worry about too much fiddling with the current right-sidebar headers if those are going to change...

Reply:
(I'm going to work right now on resolving the merge conflicts on that)

Reply:
> Would it be useful to have a template just for the <h5> that could then take a parameter for the count?

I considered that, but I want to avoid recalculating the `header_text`

What do you mean by working in text nodes?

Reply:
(I talked with Karl in DMs and we decided keeping this PR separate makes sense)

Reply:
Oh, I was just looking for `text()`